config.go

```diff
@@ -42,13 +42,13 @@ func (v Volume) String() string {
 	return ""
 }
 
-type Envs []Env
-type Env struct {
+type Tuples []Tuple
+type Tuple struct {
```

Comment:
Maybe `Pair`, `NamedValue` or `KeyValuePair` would be better name for the type.

Reply:
or `NameValuePair` bc the fields are Name` and `Value`

Reply:
And another point, `Labels` are `key/value` pairs not `name/value` pairs. So I am not 100% sure we should name it this way in the config.

Reply:
But isn't tuple ordered sequence of zero or more elements, not just two?

Reply:
I'm fine changing it. I didn't look up the exact definition when I used the `Tuple` type name. I just used it based on past experiences. But when reading the formal definitions, I see your points. (TY for the support Luke). I know `NameValuePair` is probably the most descriptive. But in the interest of fancy type names, what about `Dyad`?

Reply:
And for `Labels` do you still wan to keep the exposed config as:
```yaml
labels:
- name: role       
  value: backend
```

But if we follow kubernetes naming it should be:
```yaml
labels:
- key: role
  value: backend
```

WDYT?

Reply:
I had it that way and changed it back for consistency. I'm open to do what the hive mind thinks is best on this.

Reply:
My vote would be for key/value, so the user facing config is consistent with k8s. The internal code consistency is lower priority imho.

Reply:
As for `key vs. name` I vote for keeping it consistent within our config. Having it different is IMHO confusing for our user.

Reply:
@matejvasek just to be clear, you think that the config should look like this?
```
labels:
- name: role       
  value: backend
envs:
- name: API_KEY
  value: somesecret
```

Or this?
```
labels:
- key: role       
  value: backend
envs:
- key: API_KEY
  value: somesecret
```
Or something else?

Reply:
I like the first one more. I just prefer have it consistent within the file as is could be confusing having it different. but maybe users wouldn't mind.